staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -50,15 +50,14 @@ func WithAuthentication(handler http.Handler, auth authenticator.Request, failed
 		defer recordAuthMetrics(resp, ok, err, apiAuds, authenticationStart)
 		if err != nil || !ok {
 			if err != nil {
-				klog.Errorf("Unable to authenticate the request due to an error: %v", err)
+				klog.V(2).Infof("request (%s): unable to authenticate due to an error: %v", shortRequestInfo(req), err)
 			}
 			failed.ServeHTTP(w, req)
 			return
 		}
 
 		if !audiencesAreAcceptable(apiAuds, resp.Audiences) {
-			err = fmt.Errorf("unable to match the audience: %v , accepted: %v", resp.Audiences, apiAuds)
-			klog.Error(err)
+			klog.V(2).Infof("request (%s): unable to match the audience: %v , accepted: %v", shortRequestInfo(req), resp.Audiences, apiAuds)
```

Comment:
same question about lowering the severity of what seems like an important error condition from the configured authenticator